### PR TITLE
fix on download and style font in lesson screen

### PR DESCRIPTION
**Pull request by mouyleng**

### CODE REVIEW

File: www/js/contents/contents-services.js [hunk 1]
```diff
@@ -59,13 +49,22 @@ function ContentsServices($cordovaSQLite, $rootScope, $cordovaFileTransfer, ENDP
     }
   }
 
+  function getImageOrAudioName(content, dataUrl){
+    var name = "";
+    if(dataUrl){
+      name = dataUrl.split("/").pop();
+      mediaTransfer(content, dataUrl);
```
Comment (kakada) [inline www/js/contents/contents-services.js]: this action seems not relate with the method name, should take it off

File: www/js/contents/contents-services.js [hunk 2]
```diff
@@ -59,13 +49,22 @@ function ContentsServices($cordovaSQLite, $rootScope, $cordovaFileTransfer, ENDP
...
+    }
+    return name;
+  }
+
   function mediaTransfer(content, mediaUrl){
     var path = cordova.file.applicationStorageDirectory +  "lesson" + content.lesson_id + "/method" + content.writing_method_id + "/";
     var target = path + mediaUrl.split("/").pop();
     $cordovaFileTransfer.download(ENDPOINT.url + mediaUrl, target, {}, true).then(function (result) {
       console.log('success targetImage : ', target);
```
Comment (kakada) [inline www/js/contents/contents-services.js]: Please remove console.log if it's no needed

Comment (mouyleng) [reply www/js/contents/contents-services.js]: Fixed.

File: www/js/contents/contents-services.js (hunk 1 above)
Comment (mouyleng) [reply www/js/contents/contents-services.js]: Fixed.

**Pull request merged**